Sources/Public/Consent+PublicAPI.swift

```diff
@@ -15,10 +15,17 @@ import Foundation
 
 @objc public extension Consent {
 
-    /// Retrieves the current consent preferences stored in the Consent extension.
-    /// - Parameter completion: invoked with the current consent preferences or an `AEPError` if an unexpected error occurs or the request timed out.
+    /// Retrieves the current consent preferences stored in the Consent extension
+    /// - Parameter completion: The completion handler is invoked with the current consent preferences and an optional error
+    /// - Parameter consents:  The current consent preferences as `[String: Any]`
+    /// - Parameter error:  An `AEPError`  if an unexpected error occurs or the request timed out
+    ///
+    /// Output example:
+    /// ```
+    /// ["consents": ["collect": ["val": "y"]]]
+    /// ```
     @objc(getConsents:)
-    static func getConsents(completion: @escaping ([String: Any]?, Error?) -> Void) {
+    static func getConsents(completion: @escaping (_ consents: [String: Any]?, _ error: Error?) -> Void) {
```

Comment:
Why are we adding argument labels? We haven't done this anywhere else in the Swift SDK, and I don't find them to really add any value. From the API it's clear that this API will return the consents along with a possible error.

I think we should follow what we've done on our previous Swift APIs and not use the argument labels, especially since I don't believe they help auto-complete anything.

Reply:
I agree with @nporter-adbe. I do like the output example you've added and I think that is sufficient for users to understand what is returned in `completion`.

Reply:
@nporter-adbe @kevinlind my intention was to fix those two lines with "No description" in the quick help section, couldn't find a fix for them.. I reverted the labels.